polygon-validator-signer-balance-bot/src/agent.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { FindingType, FindingSeverity, Finding, ethers, HandleBlock, createBlockEvent } from "forta-agent";
+import { provideHandleTransaction } from "./agent";
+import { POLYGON_VALIDATOR_SIGNER_ADDRESS } from "./constants";
+
+// let MINIMUM_THRESHOLD_BIG = ethers.BigNumber.from(MINIMUM_THRESHOLD)
+describe("POLYGON-VALIDATOR-SIGNER BOT TEST SUITE", () => {
+  let handleBlock: HandleBlock;
+
+  const mockEthersProvider = {
+    getBalance: jest.fn(),
+  } as any;
+
+  const blockEvent = createBlockEvent({
+    block: { hash: "0xa", number: 1 } as any,
+  });
+
+  beforeAll(() => {
+    handleBlock = provideHandleTransaction(mockEthersProvider);
+  });
+
+  it("returns  findings if balance is above threshold", async () => {
+    const balance = ethers.BigNumber.from("2000000000000000000");
+    const formatThresholdToBigNumber = ethers.BigNumber.from("1000000000000000000");
```

Comment:
```suggestion
    const balance = ethers.BigNumber.from("2000000000000000000"); // 2 ETH
    const formatThresholdToBigNumber = ethers.BigNumber.from("1000000000000000000"); // 1 ETH
```
When using large numbers like this, it helps to include a comment stating its value for legibility for a reviewer.